test/OrchardCore.Tests/Apis/ContentManagement/DeploymentPlans/BlogPostValidateDeploymentPlanTests.cs

```diff
@@ -46,17 +46,15 @@ public async Task ShouldFailWhenAutoroutePathIsNotUnique()
                 Assert.Throws<HttpRequestException>(() => response.EnsureSuccessStatusCode());
 
                 // Confirm creation of both content items was cancelled.
-                using (var shellScope = await BlogPostDeploymentContext.ShellHost.GetScopeAsync(context.TenantName))
+                var shellScope = await BlogPostDeploymentContext.ShellHost.GetScopeAsync(context.TenantName);
```

Comment:
Interesting @jtkech was this my bad usage of `shellScope` by wrapping it in a using block (it shouldn't have one?)

Reply:
@deanmarcussen You don't need it as `UsingAsync()` will do more and at the end will also dispose the scope asynchronously, but it would have no impact to use an using block, it would just try to dispose an already disposed scope.

        /// <summary>
        /// Execute a delegate using this shell scope.
        /// </summary>
        public async Task UsingAsync(Func<ShellScope, Task> execute)
        {
            if (Current == this)
            {
                await execute(Current);
                return;
            }

            using (this)
            {
                StartAsyncFlow();

                await ActivateShellAsync();

                await execute(this);

                await BeforeDisposeAsync();

                await DisposeAsync();
            }
        }